tool/microkit/__main__.py

```diff
@@ -1258,7 +1258,7 @@ def build_system(
 
     final_cap_slot = cap_slot
 
-    ## Minting in the address space
```

Comment:
Will have to revert these kinds of changes since these comments are intentionally formatted as so.

Reply:
Seems that this is enforced and the style checker does not allow freedom in how you format comments so will have to keep these changes.